src/Cache/Engine/RedisEngine.php

```diff
@@ -65,6 +66,7 @@ class RedisEngine extends CacheEngine
         'server' => '127.0.0.1',
         'timeout' => 0,
         'unix_socket' => false,
+        'scan_count' => 10,
```

Comment:
I'm thinking we should use `scanCount`. Most configuration parameters are `camelBacked` with the exception of `unix_socket` which is the odd duck out.

Reply:
Done